presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5546,4 +5546,36 @@ public void testApproxMostFrequentWithStringGroupBy()
         assertEquals(actual1.getMaterializedRows().get(2).getFields().get(0), "c");
         assertEquals(actual1.getMaterializedRows().get(2).getFields().get(1), ImmutableMap.of("C", 2L));
     }
+
+    @Test
+    public void testMapUnionSum()
```

Comment:
Can we add a test that fails - for example on a non-numeric map value?

Reply:
Done